common4j/src/main/com/microsoft/identity/common/java/controllers/CommandDispatcher.java

```diff
@@ -210,8 +275,8 @@ public static ILocalAuthenticationResult submitAcquireTokenSilentSync(@NonNull f
     public static FinalizableResultFuture<CommandResult> submitSilentReturningFuture(@SuppressWarnings(WarningType.rawtype_warning)
```

Comment:
Consider taking tagName, executor and codemarker as input from callers.

Reply:
I was looking into it but i feel it's better to have the thread pool & executor logic inside CommandDispatcher itself ... 

We could however take methodName as input, and use that to determine which thread pool to use .... instead of using the current logic:

```
private static boolean isDeviceCodeFlowRequest(BaseCommand command) {
            return (command instanceof DeviceCodeFlowCommand
                    || command instanceof DeviceCodeFlowAuthResultCommand
                    || command instanceof DeviceCodeFlowTokenResultCommand);
        }
```